core/Core.rdf

```diff
@@ -70,7 +74,25 @@
 	</rdf:Description>
 	
 	<rdf:Description rdf:about="&bfo;BFO_0000063">
+		<iof-av:explanatoryNote xml:lang="en-US">while the scope of BFO: precedes is occurrent, for the purposes of formalization of Allen interval algebra we have added the FOL elaborations for &apos;time instant&apos; &apos;time interval&apos;, &apos;process boundary&apos; and &apos;process&apos;. For the case of other occurrents the formalization already provided by BFO (https://github.com/BFO-ontology/BFO-2020/tree/master/21838-2/pdf) should be followed.</iof-av:explanatoryNote>
+		<iof-av:firstOrderLogicAxiom>LA1: precedes(i,j) ∧ process(i) ∧ process(j) → ∃i1∃j1 (TemporalInterval(i1)∧TemporalInterval(j1)∧occupiesTemporalRegion(i,i1)∧occupiesTemporalRegion(j,j1) ∧ precedes(i1,j1))</iof-av:firstOrderLogicAxiom>
+		<iof-av:firstOrderLogicAxiom>LA2: precedes(i,j) ∧ ProcessBoundary(i) ∧ ProcessBoundary(j) → ∃i1∃j1 (TemporalInstant(i1)∧TemporalInstant(j1) ∧ occupiesTemporalRegion(i,i1) ∧ occupiesTemporalRegion(j,j1) ∧ precedes(i1,j1))</iof-av:firstOrderLogicAxiom>
+		<iof-av:firstOrderLogicAxiom>LA3: precedes(i,j) ∧TemporalInstant(i) ∧TemporalInterval(j) → ∃j1(hasFirstInstant(j,j1)∧ precedes(i,j1))</iof-av:firstOrderLogicAxiom>
+		<iof-av:firstOrderLogicAxiom>LA4: precedes(i,j) ∧ TemporalInterval(i) ∧ TemporalInstant(j) → ∃i1(hasLastInstant(i,i1) ∧ precedes(i1,j))</iof-av:firstOrderLogicAxiom>
+		<iof-av:firstOrderLogicAxiom>LA5: precedes (i,j) ∧TemporalInstant (i) ∧ TemporalInstant(j) → ∃k∃l(hasValueExpressionAtAllTimes(i,k) ∧ hasValueExpressionAtAllTimes(j,l) ∧ ∃ v1,v2(hasDateTimeInstantValue(k,v1) ∧ hasDateTimeInstantValue(l,v2) ∧ (v1 &lt; v2)))</iof-av:firstOrderLogicAxiom>
```

Comment:
Multiple quantifiers.

Reply:
> can you maybe explain a little bit what you mean by multiple quantifiers?

Similar issue as in https://github.com/iofoundry/ontology/pull/76#discussion_r1498899426.

Reply:
do you mean that there were multiple ways of specifying the quantifiers within a single FOL? Should be fixed with this latest commit, if its something else please provide more details